ansible_ai_connect/ai/api/tests/test_views.py

```diff
@@ -220,6 +228,32 @@ def login(self):
         self.client.login(username=self.username, password=self.password)
 
 
+class WisdomServiceAPITestCaseBaseOIDC(WisdomServiceAPITestCaseBase):
+    """This should ultimately replace WisdomServiceAPITestCaseBase"""
+
+    def setUp(self):
+        self._inibate_create_user = True
```

Comment:
Is this a typo?

`inibate` vs `inhibit`?

Reply:
ahah